pkg/util/logging/log_counter_hook.go

```diff
@@ -47,7 +51,20 @@ func (h *LogCounterHook) Fire(entry *logrus.Entry) error {
 	defer h.mu.Unlock()
 
 	h.counts[entry.Level]++
-
+	if h.entries[entry.Level] == nil {
+		h.entries[entry.Level] = &persistence.Result{}
+	}
+	ns, isPresent := entry.Data["namespace"]
+	if isPresent {
+		h.entries[entry.Level].Add(ns.(string), errors.New(entry.Message))
+	} else {
+		errorField, isErrorFieldPresent := entry.Data["error"]
+		if isErrorFieldPresent {
+			h.entries[entry.Level].Merge(&persistence.Result{Velero: []string{entry.Message + ": " + errorField.(error).Error()}})
+		} else {
+			h.entries[entry.Level].Merge(&persistence.Result{Velero: []string{entry.Message}})
```

Comment:
Similar to above, using AddVeleroError and let persistence.Result append to the Velero entry.

Reply:
Make sense, will refactor.